demo/vl-data-table.html

```diff
@@ -400,5 +400,181 @@ <h2>Data table Zebra</h2>
   </div>
```

Comment:
Best de demo pagina zoveel als mogelijk beperken tot die van Webuniversum. Lijkt me niet echt nodig om voor elke variant een demo te voorzien.

Reply:
dan kan ik niet testen of al mijn methodes op mijn page object werken ...